packages/next/src/client/form.tsx

```diff
@@ -101,6 +120,11 @@ export default function Form({
             '  `router.replace()` - https://nextjs.org/docs/app/api-reference/functions/use-router#userouter\n'
         )
       }
+      if (prefetch !== undefined) {
```

Comment:
This will always log since we initialize prefetch with a `null` default value

https://github.com/vercel/next.js/blob/b5d7757e7a36b88a5c97655c8fa5582555ec40f3/packages/next/src/client/form.tsx#L56

null !== undefined --> true

Reply:
good catch, it was late when i wrote this 😅